scipy/sparse/csgraph/_laplacian.py

```diff
@@ -112,6 +112,9 @@ def laplacian(
     The normalization is symmetric, making the normalized Laplacian also
     symmetric if the input csgraph was symmetric.
 
+    The algorithmic complexity is for a graph with V
+    vertices is O(V^2).
```

Comment:
Thanks for tackling this, I wonder if these should be monospace
```suggestion
    vertices is ``O(V^2)``.
```